arbeitszeit_flask/mail_service.py

```diff
@@ -50,7 +49,4 @@ def send_message(
         sender: str,
     ) -> None:
         msg = Message(subject=subject, recipients=recipients, html=html, sender=sender)
-        try:
-            mail.send(msg)
-        except Exception:
-            raise CannotSendEmail
+        mail.send(msg)
```

Comment:
Does sending a mail message never raise an exception from somewhere in the process?

Reply:
OK, good point.